sdk/include/opentelemetry/sdk/logs/batch_log_processor.h

```diff
@@ -77,21 +77,21 @@ class BatchLogProcessor : public LogProcessor
                              const BatchLogProcessorOptions &options);
 
   /** Makes a new recordable **/
-  std::unique_ptr<Recordable> MakeRecordable() noexcept override;
+  virtual std::unique_ptr<Recordable> MakeRecordable() noexcept override;
```

Comment:
We should not add `virtual` keyword for overried methods. Clang will warning this.

Reply:
ok, will make the change.